README.md

```diff
@@ -1,4 +1,329 @@
+
+Name | Description | Example
+--- | --- | ---
+accessibilityId, id, name | These all strategies are mapped to the same Mac2 driver ByIdentifier lookup strategy. The locator matches the passed value with element's `identifer` attribute case-sensitively. | `MobileBy.accessibilityId("identifier")`, `By.id("identifier")`, `By.name("identifier")`
+className | Class name uses stringified element types for lookup | `By.className("XCUIElementTypePopUpButton")`
+predicate | Lookup by predicate is natively supported by XCTest and is as fast as previous lookup strategies. This lookup strategy could only use the supported [element attributes](#element-attributes). Unknown attribute names would throw an exception. Check [NSPredicate cheat sheet](https://academy.realm.io/posts/nspredicate-cheatsheet/) for more details on how to build effective and flexible locators. | `MobileBy.iOSNsPredicateString("elementType == 2 AND label BEGINSWITH 'Safari'")`
+classChain | This strategy is a combination of Xpath flexibility and fast predicate lookup. Prefer it over Xpath unless there is no other way to build the desired locator. Visit [Class Chain Construction Rules tutorial](https://github.com/facebookarchive/WebDriverAgent/wiki/Class-Chain-Queries-Construction-Rules) to get more knowledge on how to build class chain locators. | `MobileBy.iOSClassChain("**/XCUIElementTypeRuler[$elementType == 72 AND value BEGINSWITH '10'$]"`)
+xpath | For elements lookup Xpath strategy the driver uses the same XML tree that is generated by page source API. Only Xpath 1.0 is supported (based on xmllib2). | `By.xpath("//XCUIElementTypePopUpButton[@value="Regular" and @label="type face"]")`
+
+Check the [integration tests](/test/functional/find-e2e-specs.js) for more examples on different location strategies usage.
+
+
+## Platform-Specific Extensions
+
+Beside of standard W3C APIs the driver provides the following custom command extensions to execute platform specific scenarios:
+
+### macos: click
+
+Perform click gesture on an element or by relative/absolute coordinates
+
+#### Arguments
+
+Name | Type | Required | Description | Example
+--- | --- | --- | --- | ---
+element | string | if `x` or `y` are unset | Unique identifier of the element to perform the click on. Either this property or/and x and y must be set. If both are set then x and y are considered as relative element coordinates. If only x and y are set then these are parsed as absolute coordinates. | 21045BC8-013C-43BD-9B1E-4C6DC7AB0744
+x | number | if `y` is set or `element` is unset | click X coordinate | 100
+y | number | if `y` is set or `element` is unset | click Y coordinate | 100
+keyModifierFlags | number | no | if set then the given key modifiers will be applied while click is performed. See the official documentation on [XCUIKeyModifierFlags enumeration](https://developer.apple.com/documentation/xctest/xcuikeymodifierflags) for more details | 1 << 1 `logical or` 1 << 2
```

Comment:
`1 << 1` logical or `1 << 2`? (for example of keyModifierFlags)

Reply:
there is no error there